forch/forchestrator.py

```diff
@@ -37,10 +37,29 @@
 _BEHAVIORAL_CONFIG_DEFAULT = 'faucet.yaml'
 _SEGMENTS_VLAN_DEFAULT = 'segments-to-vlans.yaml'
 _DEFAULT_PORT = 9019
-_FAUCET_VARZ_HOST = '127.0.0.1'
-_FAUCET_PROM_PORT = 9302
-_GAUGE_VARZ_HOST = '127.0.0.1'
-_GAUGE_PROM_PORT = 9303
+_FAUCET_HOST_DEFAULT = '127.0.0.1'
+_FAUCET_PROM_PORT_DEFAULT = 9302
+_GAUGE_HOST_DEFAULT = '127.0.0.1'
+_GAUGE_PROM_PORT_DEFAULT = 9303
+
+_TARGET_FAUCET_METRICS_DEFAULT = (
```

Comment:
these aren't defaults (what else might they be?) -- they are actuals (this and gauge).

Reply:
Removed DEFAULT